extensions/pgx_pgmq/Cargo.toml

```diff
@@ -25,7 +25,7 @@ pg_test = []
 [dependencies]
 pgx = "0.7.1"
 serde = "1.0.152"
-pgmq = { path = "../../crates/pgmq", version = ">=0.4.0, <1.0.0" }
+pgmq = ">=0.4.0, <1.0.0"
```

Comment:
undecided if this is a good idea - it's because if we are building the extension with Docker, then there is some hassle to use the relative pathing

Reply:
Ah ya, good point.

Reply:
TODO: review this

Reply:
It's possible to copy the crate before building the container. Which do you think? I don't mind either way.

Reply:
lets point it to crates.io like you have it